WorkloadManagementSystem/Agent/StalledJobAgent.py

```diff
@@ -568,4 +567,20 @@ def __failSubmittingJobs(self):
 
     return S_OK()
 
+  def __sendKillCommand(self, job):
+    """Send a kill signal to the job such that it cannot continue running.
+
+    :param int job: ID of job to send kill command
+    """
+    ownerDN = self.jobDB.getJobAttribute(job, 'OwnerDN')
+    ownerGroup = self.jobDB.getJobAttribute(job, 'OwnerGroup')
+    if ownerDN['OK'] and ownerGroup['OK']:
+      wmsClient = WMSClient(useCertificates=True, delegatedDN=ownerDN['Value'], delegatedGroup=ownerGroup['Value'])
+      resKill = wmsClient.killJob(job)
+      if not resKill['OK']:
+        self.log.error("Failed to send kill command to job", "%s: %s" % (job, resKill['Message']))
+    else:
+      self.log.error("Failed to get ownerDN or Group", "%s, %s" %
```

Comment:
I would add the JobID in this error message.

Reply:
Done